explorer/testdata/assert/fail_convert.carbon

```diff
@@ -5,6 +5,7 @@
 // AUTOUPDATE
 // RUN: %{not} %{explorer-run}
 // RUN: %{not} %{explorer-run-trace}
+// CHECK:STDERR: RUNTIME ERROR: {{.*}}/explorer/data/prelude.carbon:{{.*}}: "Fail"
```

Comment:
Could this be directly above the line that fails, like before?

Reply:
It was above the line that failed before because it was associated with a line in this file. `Assert` associates with the prelude file, so this is the automatic location.

If I change this manually, it would mean any future edits would need to be done manually as well which seems undesirable.